bakery/service.go

```diff
@@ -18,11 +19,14 @@ import (
 
 var logger = loggo.GetLogger("bakery")
 
+var uuidGen = fastuuid.MustNewGenerator()
```

Comment:
Even so, macaroon-bakery is just a library, it should be up to the app to decide, but at least it would happen on startup I guess. One concern I have is that depending on the un knowable logging strategy of said app this panic could be logged without any context of where it came from. Why not just do it in NewService?